src-plugins/medClut/medClutToolBox.cpp

```diff
@@ -450,43 +440,37 @@ void medClutToolBox::exportMeshWithLUT()
 {
     vtkMetaSurfaceMesh * mesh = vtkMetaSurfaceMesh::New();
     mesh->SetDataSet(d->viewDataSet->value(d->view));
-    if (!mesh)
-        return;
-    
-    dtkSmartPointer<medAbstractData> dataToExport = medAbstractDataFactory::instance()->createSmartPointer("vtkDataMesh");
-    dataToExport->setData(mesh);
-
-    vtkPointSet * pointSet = vtkPointSet::SafeDownCast(mesh->GetDataSet());
+    if (mesh)
```

Comment:
It's useless to check the success of an allocation because in the (extremely rare) case where it will actually fail an exception will be thrown, so the check will not be reached (also the line before would have crashed anyways). Also, in the case where it fails, the whole system is probably about to crash so trying to take any form of coherent action in the program is meaningless.